verticapy/tests_new/core/vdataframe/test_math.py

```diff
@@ -0,0 +1,885 @@
+"""
+Copyright  (c)  2018-2023 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+import re
+import datetime
+import pandas as pd
+import numpy as np  # pylint: disable=unused-import
+from scipy.stats import (
+    median_abs_deviation,
+    jarque_bera,
+)  # pylint: disable=unused-import
+import pytest
+from verticapy.core.tablesample.base import TableSample
+from verticapy.tests_new.core.vdataframe import REL_TOLERANCE, ABS_TOLERANCE
+from verticapy.tests_new import functions, AggregateFun
+
+
+class TestMath:
+    """
+    test class for Text functions test
+    """
+
+    @pytest.mark.parametrize(
+        "input_type, columns",
```

Comment:
I think we should have a really simple test for this. Titanic is probably not the most ideal dataset as it does not have negative values. 

You can create a very simple dataset like:
vp.vDataFrame({"col1":[-1, 1, 2], "col2":[-10, -2.0, 3]})

Also we do not need to try a lot of combinations as they will add time to our code test. SImply one vDataFrame with two columns and one vcolumn should be enoush.

Reply:
I am using normalize function on numeric columns (i.e. focus_columns) to scale data.
example -
titanic_vdf = titanic_vd_fun[focus_columns].normalize().fillna(0)
1        -1.9502503129565278  
2      -0.010561423955012732  
3       -0.35693443984814044  
4         0.6129100046526171  
5         2.8296973063686344 

Test combinations are basic scenarios (one column, multiple columns and no columns for vDataFrame). it's based on input of abs function for vDataFrame and vcolumn.